src/components/Modal/TransactionModal/TransactionIsPending.tsx

```diff
@@ -20,28 +18,24 @@ const TransactionIsPending: FC<TransactionIsPendingProps> = ({
 }) => {
   return (
     <>
-      <ModalHeader>Confirm (pending)</ModalHeader>
-      <ModalCloseButton />
-      <ModalBody>
-        <InfoBox py={12} variant="modal">
-          <ThresholdSpinner />
-          <BodyLg align="center" mt={8}>
-            {pendingText}
-          </BodyLg>
-        </InfoBox>
+      <ModalBody as={Flex} flexFlow="column" align="center" p={6}>
+        <Spinner />
+        <BodyLg mt={10} mb={9}>
+          {pendingText}
+        </BodyLg>
       </ModalBody>
-      <ModalFooter justifyContent="center">
+      <ModalFooter p={6} justifyContent="center">
         <BodySm>
           <ViewInBlockExplorer
             text="View"
             id={transactionHash}
             type={ExplorerDataType.TRANSACTION}
-          />{" "}
-          transaction on Etherscan
+          />
+          &nbsp;transaction on Etherscan
```

Comment:
Non-blocking: 

No need to use `&nbsp;` here

Reply:
Ref commit: 25a9c7c26a385ee187058fe261fcb4d18a28048a